libs/dh/admin/feature-user-management/src/lib/tabs/users-tab/dh-users-tab.component.ts

```diff
@@ -124,7 +124,7 @@ export class DhUsersTabComponent {
   readonly hasGeneralError$ = this.store.hasGeneralError$;
 
   readonly initialStatusFilter$ = this.store.initialStatusFilter$;
-
+  isInviteUserModalVisible = false;
```

Comment:
Question: I expect this is necessary because you want the content to be "lazy"? I would suggest that we talk about this in the guild since this is going to be tedious if we should do this every time.

Reply:
It because, we want the component to be removed / unmount / reset when we close the modal